pmonitor/pmon

```diff
@@ -21,7 +21,7 @@ while [ "$1" != "" ]; do
   shift;
 done;
 
-export PID_DIR=${WORKDIR}/pid_monitor
+export PID_DIR=${WORKDIR}/perftools-setup/pid_monitor
```

Comment:
Can you please check the path where we extract the pid_monitor in our setup scripts:  https://github.com/josiahsams/perftools-setup/blob/master/install.sh#L80